src/app/admin/modules/program-management/admin-events/components/admin-campus-event-list/admin-campus-event-list.component.html

```diff
@@ -5,39 +5,8 @@
       <h4 class="card-title">项目名称:{{ program.name }}</h4>
       <p class="card-category">开设单位:{{ program.department }}</p>
     </div>
-    <div class="card-body">
-      <div class="float-left">
-        <button mat-raised-button color="primary" (click)="navigateToProgramDetail()">查看项目详情</button>
-      </div>
-      <div class="float-right">
-        <button mat-raised-button color="primary" (click)="navigateToCreateForm()">创建新活动</button>
-      </div>
-      <div class="loading-shade" *ngIf="isLoadingResults">
-        <mat-spinner></mat-spinner>
-      </div>
-      <div class="table-responsive">
-        <table class="table table-hover text-center">
-          <thead class="text-primary">
-            <tr>
-              <th scope="col">名字</th>
-              <th scope="col">时间</th>
-              <th scope="col">地点</th>
-              <th scope="col">总时长</th>
-              <th scope="col">参与者人数</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr *ngFor="let item of results" (click)="navigateToDetail(item)">
-              <td scope="col">{{ item.name }}</td>
-              <td scope="col">{{ item.time | date:'y.M.d' }}</td>
-              <td scope="col">{{ item.location }}</td>
-              <td scope="col">{{ item.num_hours | number:"0.1-1" }} </td>
-              <td scope="col">{{ item.num_participants }}</td>
-            </tr>
-          </tbody>
-        </table>
-        <mat-paginator [length]="resultsLength" [pageSize]="pageSize" [disabled]="isLoadingResults"></mat-paginator>
-      </div>
+    <div class="card-body" *ngIf="program">
```

Comment:
Since you check the existence of `program` multiple times at the same level, how about move the check to parent level, so you just need to check it at one place.